packages/components/src/font-size-picker/next.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import { FontSizeControl } from '@wp-g2/components';
```

Comment:
> Could we bring the code of FontSizeControl to Gutenberg

To do that, we would need (at minimum) all of the Components (and systems) that make up that control.
I've included a screenshot below of a high-level overview. Required components are in the yellow squares:

<img width="1639" alt="Screen Shot 2020-12-10 at 2 51 27 PM" src="https://user-images.githubusercontent.com/2322354/101822362-34acb900-3af7-11eb-9ee0-b2e88b43d056.png">

Link to Miro board (source of screenshot)
https://miro.com/app/board/o9J_khVkF9A=/?moveToWidget=3074457351837142359&cot=14

---

Adding the systems and components to build up to FontSizeControl was the original proposal.
However, @gziolo suggested we could try consuming the `@wp-g2/components` package directly to expedite integration, which I think is a good idea (assuming important things like bundling aren't severely impacted)

Reply:
Can't we add FontSizeControl component (just this component)  and use its dependencies from the package? The idea is that this will allow us to review the component properly in isolation and step by step we'd do the same for all its dependencies.